### DISCUSSION TITLE
Partner Center Integration stripping characters from publisher name

### DISCUSSION THREAD

**Discussion by Ghelwig** [codemagic-ci-cd/codemagic-docs#2159] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Create a Windows Build and Enable Distribution thorough the Microsoft Partner Center
2. under Publisher name, add a comma to the name, for instance my publisher name in my Partner Center looks like 'Company, Inc.' so that is what I added in code magic
3. look at the build progress, and you can see that the command that sets the publisher display name for the MSIX app says '--publisher-display-name 'Company Inc.' '
4. This will make all automated builds fail since the publisher name no longer matches what is needs to be in the partner center

Is there some sort of work around where we can enter commas in the publisher name or even bypass the field completely since it is the correct name in our pubspec.yaml msix configuration? 

### Expected results

The publisher name field does not strip out text so that it can match the Partner Center.

### Actual results

The publisher name strips out certain characters making it impossible to user with the Partner Center integration.

**Comment by dtrdic** [comment]
Solved over Intercom
